RAMP/ramp.py

```diff
@@ -96,8 +97,11 @@ def unpack(bundle):
 @click.option('--dependencies', callback=jsons_str_tuple_to_jsons_tuple, help='list of module dependencies: <name, uri, sha256>')
 @click.option('--output', '-o', default='module.zip', help='output file name')
 @click.option('--print-filename-only', '-P', is_flag=True, default=False, help="Print package path, but don't generate file")
+@click.option('--packname-file', default=None, help="Print package name to the file")
```

Comment:
"Print package name to the file" ?
Not sure I understand what the packname-file does

Reply:
Similar to this, from docker documentation:
```
--cidfile="": Write the container ID to the file
```
It writes the actual package name (that can contain variables) to the given file.
Replace 'print' with 'write'?

Reply:
yes please.